release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -925,6 +1008,54 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-monitoring-bug-fixes"]
 ==== Monitoring
 
+* Before this update, dashboards in the {product-title} web console that contained queries using a container label for `container_fs*` metrics returned no data points because the container labels had been dropped due to high cardinality. 
+This update resolves the issue, and these dashboards now display data as expected.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2037513[*BZ#2037513*])
+
+* Before this update, the `prometheus-operator` component allowed any time value for `ScrapeTimeout` in the config map. 
+If you set `ScrapeTimeout` to a value greater than the `ScrapeInterval` value, Prometheus would stop loading the config map settings and fail to apply all subsequent configuration changes.
+With this update, if the `ScrapeTimeout` value specified is greater than the `ScrapeInterval` value, the system logs the settings as invalid, but continues loading the other config map settings.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2037762[*BZ#2037762*])
+
+* Before this update, in the *CPU utilisation* dashboard in the {product-title} web console, the formula used to calculate CPU utilisation of a node could incorrectly display invalid negative values.
```

Comment:
CPU Utilisation

Reply:
Fixed. Also reworded to add the name of the dashboard:

`Before this update, in the *CPU Utilisation* panel on the *Kubernetes / Compute Resources / Cluster* dashboard`